### PR TITLE
Upgrade to Riot-desktop 1.6.0

### PR DESCRIPTION
**Pull request by SISheogorath**
Upgrading to the new release 1.6.0 with all needed dependency changes to use the new riot-desktop package, build the required dependencies and offer full-featured cross-signing with search indexing etc.

### CODE REVIEW

File: im.riot.Riot.json [hunk 1]
```diff
@@ -23,17 +23,70 @@
...
+                "sha256": "c43cb0c1518ce42b00e7c8f6eaddd5195c53a98f94adc717234a65cbcfd3f96a"
+                }
+            ]
+        },
+        {
+            "name": "sqlcipher",
+            "rm-configure": true,
+            "config-opts": [
+                "--enable-tempstore=yes",
+                "--disable-tcl"
```
Comment (Erick555) [inline im.riot.Riot.json:49]: Do you still need tcl if it's disabled here?

File: im.riot.Riot.json [hunk 2]
```diff
@@ -23,17 +23,70 @@
...
+            ],
+            "build-options": {
+                "env": {
+                "CFLAGS": "-g -O2 -DSQLITE_HAS_CODEC",
```
Comment (Erick555) [inline im.riot.Riot.json:53]: Is full `CFLAGS` env override needed? Maybe you can just pass `-DSQLITE_HAS_CODEC` in `cflags` variable which will append it to default set.

File: im.riot.Riot.json [hunk 3]
```diff
@@ -23,17 +23,70 @@
...
+                "LDFLAGS": "-lcrypto"
```
Comment (Erick555) [inline im.riot.Riot.json:54]: Same as `CFLAGS`

File: im.riot.Riot.json (hunk 1 above)
Comment (SISheogorath) [reply im.riot.Riot.json:49]: Sadly yes, because the `./configure` step requires `tclsh`

File: im.riot.Riot.json (hunk 3 above)
Comment (SISheogorath) [reply im.riot.Riot.json]: Done

File: im.riot.Riot.json (hunk 2 above)
Comment (SISheogorath) [reply im.riot.Riot.json]: Done

**Pull request merged**